### ISSUE TITLE
Results about test sets

### ISSUE THREAD

**Issue by 1090118235** [cwmok/LapIRN#21]
Dear @cwmok ，
I added a function to calculate dice score on the basis of code Test_LapIRN_disp.py. I selected 001 data as atlas and 100 data as test set (excluding test set). But the calculation result is very poor, only 0.41. Why?

**Comment by cwmok** [comment]
Hi @1090118235,

Which dataset you are using? Did you train your own model or use the pre-trained model?

**Comment by 1090118235** [comment]
> Hi @1090118235,
> 
> Which dataset are you using? Did you train your own model or use the pre-trained model?
Thank you for your reply，I use OASIS dataset ，and  I tested on my own model and pre training model，the results are 0.4 and 0.41 respectively.

**Comment by cwmok** [comment]
Hi @1090118235,

If you want to apply your dataset with our pre-trained model, remember to pre-process your data in the same format as our data, as shown in https://github.com/cwmok/LapIRN/tree/master/Data. 

Some users reported a similar issue regarding training the model. The problem can be solved by re-installing the `Pytorch 1.3.0 - 1.7.0` with the proper CUDA version. Mismatching the CUDA version of Pytroch with your GPU driver may cause a similar problem.

Finally, if you still cannot figure out the issue, you may send the code, your own model, 1 pair of images from the OASIS dataset and the deformed moving image here. I can do the parameter check for you.
